apps/maeilmail/src/components/setting/apis/getMailFrequency.ts

```diff
@@ -1,8 +1,12 @@
 import mainClient from '@/common/apis/client/mainClient';
 import { pathGenerator } from '@/common/apis/constants/routes';
 
+interface MailFrequency {
+  frequency: 'daily' | 'weekly';
+}
```

Comment:
common/types/mail.ts`에 위치한 MailFrequency를 import해서 사용하면 될 것 같아요!
```suggestion
interface GetMailFrequencyResponse {
  frequency: 'daily' | 'weekly';
}
```

Reply:
미쳐 확인하지 못했네요. 수정해두겠습니다!